deploy/docker-compose.yml

```diff
@@ -0,0 +1,79 @@
+version: "3.9"
+
+services:
+  nginx:
+    image: nginx:latest
+    ports:
+      - "127.0.0.1:${INTERNAL_HOST_PORT:-1235}:80"
+    restart: always
+    depends_on:
+      - s-pipes-engine
+      - s-pipes-editor-rest
+      - db-server
+      - s-pipes-editor-ui
+    environment:
+      NGINX_ENVSUBST_OUTPUT_DIR: "/etc/nginx"
+      APP_ORIGIN: "${PUBLIC_ORIGIN:-http://localhost:${INTERNAL_HOST_PORT:-1235}}"
+      SERVICE_URL: "${SPIPES_EDITOR_REST:-http://s-pipes-editor-rest:18115}"
+    volumes:
+      - ./nginx/nginx.conf:/etc/nginx/templates/nginx.conf.template:ro
+      - ./nginx/error.html:/usr/share/nginx/html/error.html:ro
+
+  s-pipes-editor-ui:
+    image: "ghcr.io/kbss-cvut/s-pipes-editor-ui/s-pipes-editor-ui:latest"
+    expose:
+      - "80"
+    depends_on:
+      - s-pipes-editor-rest
+    environment:
+      SERVICE_URL: "${SPIPES_EDITOR_REST:-http://s-pipes-editor-rest:18115}"
+
+  s-pipes-editor-rest:
+    #    <<: *s-pipes-editor-rest-dev-port
+    image: "ghcr.io/kbss-cvut/s-pipes-editor/s-pipes-editor:latest"
+    expose:
+      - "18115"
+    depends_on:
+      - s-pipes-engine
+      - db-server
+    environment:
+      SCRIPTPATHS: "${SCRIPTPATHS:-${PWD}/../../s-pipes-modules;${CUSTOM_SCRIPT_PATHS:-${PWD}/../../s-pipes/doc/examples}}"
+      SCRIPTRULES: "${SCRIPTRULES:-${PWD}/../../s-pipes-editor/src/main/resources/rules}"
+      ENGINEURL: "${SPIPES_ENGINE:-http://s-pipes-engine:8080/s-pipes}/"
+      RDF4J_REPOSITORYURL: "${RDF4J_SERVER_URL:-http://db-server:7200}/repositories"
+      RDF4J_REPOSITORYNAME: "${RDF4J_REPOSITORYNAME:-s-pipes-hello-world}"
+      RDF4J_PCONFIGURL: "${RDF4J_PCONFIGURL:-./s-pipes-engine/config.ttl}"
+    volumes:
+      - /tmp:/tmp
+      - ${SHARED_ROOT:-/home}:${SHARED_ROOT:-/home}
+      - /usr/local/tomcat/temp/:/usr/local/tomcat/temp/
+
+  s-pipes-engine:
+    image: "ghcr.io/kbss-cvut/s-pipes/s-pipes-engine:latest"
+    #    container_name: s-pipes-engine
+    ports:
+      - "8080:8080"
```

Comment:
We should not need this, but can have:
   expose:
      - "8080"

Reply:
done